akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -469,6 +471,8 @@ final class Broadcast[T](private val outputPorts: Int, eagerCancel: Boolean) ext
 
 object Partition {
 
+  case class PartitionOutOfBoundsException(msg:String) extends RuntimeException(msg) with NoStackTrace
```

Comment:
It does not compare well in https://github.com/israel/akka/blob/israel-%2319591/akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphPartitionSpec.scala#L179